web/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, routes } from '@redwoodjs/router'
+import { MetaTags } from '@redwoodjs/web'
+
+import Button from 'src/components/Button/Button';
+
+import LandingBG from '../../../public/landingBg.svg';
+import WelcomeIMG from '../../../public/welcomeImg.svg';
+
+const LandingPage = () => {
+  return (
+    <>
+      <MetaTags title="Landing" description="Landing page" />
+
+      <div className='relative w-full'>
+      <div className='max-h-screen overflow-hidden z-[-1]'>
+        <LandingBG className='' />
```

Comment:
Since the image is not filling the container vertically, we might see something like this: 
![image](https://user-images.githubusercontent.com/18635074/190530420-457cdbbf-4485-4d20-971b-8cdc2ad5b7b7.png)